releases-istio/apps/products-search-helm-release.yaml

```diff
@@ -0,0 +1,73 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: nordmart-dev-products-seaarch
+  namespace: nordmart-dev-apps
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.deployment: regexp:^([0-9]+.[0-9]+.[0-9]+-PR-[0-9]{2}-[0-9]+-SNAPSHOT)$
+    sidecar.istio.io/inject: "true"
+spec:
+  releaseName: nordmart-dev-products-seaarch
+  chart:
+    repository: https://stakater.github.io/stakater-charts/
+    name: application
+    version: 0.0.12
+  values:
+    applicationName: "products-search"
+    deployment:
+      podLabels:
+        app: catalog
+      volumes: {}
+      image:
+        repository: stakater/stakater-nordmart-products-search
+        tag: v0.0.1
+      fluentdConfigAnnotations:
+        regex: /^(?<time>\\d+(?:-\\d+){2}\\s+\\d+(?::\\d+){2}\\.\\d+)\\s*(?<level>\\S+)
+          (?<pid>\\d+) --- \\[(?<thread>[\\s\\S]*?)\\] (?<class>\\S+)\\s*:\\s*(?<message>[\\s\\S]*?)(?=\\g<time>|\\Z)/
+        regexFirstLine: /^\\d+(?:-\\d+){2}\\s+\\d+(?::\\d+){2}\\.\\d+/
+        timeFormat: "%Y-%m-%d %H:%M:%S.%L"
+      env:
+      - name: ELASTICSEARCH_PASSWORD
+        valueFrom:
+          secretKeyRef:
+            name: elasticsearch-secret
+            key: elasticsearch_password
+
+      - name: ELASTICSEARCH_HOST
+        value: "elasticsearch-svc.nordmart-dev-apps"
+      - name: ELASTICSEARCH_PORT
+        value: "9200"
+
+      probes:
+        readinessProbe:
+          failureThreshold: 3
+          periodSeconds: 60
+          successThreshold: 1
+          timeoutSeconds: 1
+          initialDelaySeconds: 10
+          httpGet:
+            path: /actuator/health
+            port: 8080
+        livenessProbe:
+          failureThreshold: 3
+          periodSeconds: 60
+          successThreshold: 1
+          timeoutSeconds: 1
+          initialDelaySeconds: 10
+          httpGet:
+            path: /actuator/health
+            port: 8080
```

Comment:
I don't see actuator being used anywhere in https://github.com/stakater-lab/stakater-nordmart-search
Either add dependency and ensure that it polls for kafka and elasticsearch as well
```
dependency>
<groupId>org.springframework.boot</groupId>
<artifactId>spring-boot-starter-actuator</artifactId>
</dependency>
```

Or, create a separate endpoint for `/health` and handle it there

Reply:
Added spring-boot-starter-actuator

Reply:
Hey, please check and ensure that the health point checks for kafka and elasticsearch as well. Actuator should be able to pick it up from `Configuration` but it would be great if you can double check this.